harness/determined/deploy/gcp/gcp.py

```diff
@@ -128,6 +143,16 @@ def terraform_apply(configs: Dict, env: Dict, variables_to_exclude: List) -> Non
 
 
 def terraform_output(configs: Dict, env: Dict, variable_name: str) -> Any:
+    if configs.get("tf_state_gcs_bucket_name"):
```

Comment:
I'm not clear on why this is necessary. The outputs still get printed at the end when you're using a remote state store. Isn't that how we're grabbing them in the default case?

Reply:
we need to grab them to do a health check, i.e. check if master is up by doing a http request.